lib/vault/encrypted_model.rb

```diff
@@ -41,31 +42,22 @@ module ClassMethods
       #   a proc to encode the value with
       # @option options [Proc] :decode
       #   a proc to decode the value with
+      # @option options [Hash, String] :transform_secret
```

Comment:
Do we have tests/examples of this being passed in as a string? The spec tests seems to all use hash input values for this param.

Reply:
Removed